vxyowsup/whatsapp.py

```diff
@@ -55,17 +62,28 @@ def setup_transport(self):
     def teardown_transport(self):
         print "Stopping client ..."
         self.stack_client.client_stop()
+        yield self.redis._close()
         yield self.client_d
```

Comment:
The `yield redis._close()` should come after `yield self.client_d` in case the WhatsAppInterface tries to do anything that accesses redis before it finishes stopping (e.g. an ack might arrive from WhatsApp after we've called `client_stop()` but before the client has finished stopping).